app/components/avo/index/table_row_component.html.erb

```diff
@@ -1,4 +1,8 @@
 <%# hover:z-[21] removed from tr class to solve flickering actions component on row controls and z-20 changed to z-21%>
+<%
+  resource_controls_placement = @resource.controls_placement || Avo.configuration.resource_controls_placement
+%>
```

Comment:
Let's extract this logic to a concern, something like

```
def controls_placement
  @resource.controls_placement || Avo.configuration.resource_controls_placement
end

def resource_controls_on_the_right?
  controls_placement == :right
end

def resource_controls_on_the_left?
  controls_placement == :left
end
```

Then include the concern on the components that uses this logic.

Let's also ensure that the original methods `resource_controls_on_the_right?` and `resource_controls_on_the_left?` are deleted if they are no longer used.